bears/python/PEP8Bear.py

```diff
@@ -31,14 +31,18 @@ def run(self, filename, file,
         Detects and fixes PEP8 incompliant code. This bear will not change
         functionality of the code in any way.
 
-        :param max_line_length:   Maximum number of characters for a line.
-                                  When set to 0 allows infinite line length.
-        :param indent_size:       Number of spaces per indentation level.
-        :param pep_ignore:        A list of errors/warnings to ignore.
-        :param pep_select:        A list of errors/warnings to exclusively
-                                  apply.
-        :param local_pep8_config: Set to true if autopep8 should use a config
-                                  file as if run normally from this directory.
+        :param max_line_length:
+            Maximum number of characters for a line
```

Comment:
Missing full stop at the end?

Reply:
I think thats only a one sentence  isn't it??? :thinking:

Reply:
@li-boxuan is obviously correct here.
The previous docstring had a full stop here.
And you have a capital letter for the first word on the next line.
Be critical of your own work, especially when you have been given a good review already which has found things which should be improved.
We are all lazy and forget things - that is why we rely on each other to see the things we missed.  Repeat your reviewers.